numpy/_core/fromnumeric.py

```diff
@@ -925,10 +925,11 @@ def sort(a, axis=-1, kind=None, order=None):
        is actually used, even if 'mergesort' is specified. User selection
        at a finer scale is not currently available.
 
-    All the sort algorithms make temporary copies of the data when
-    sorting along any but the last axis.  Consequently, sorting along
-    the last axis is faster and uses less space than sorting along
-    any other axis.
+    For performance, ``sort`` makes a temporary copy if needed to make the data
```

Comment:
This is fine, but to nitpick, I think the "for performance" doesn't add anything.  We could just say something in the direction of:

Sorting always operates on contiguous memory and makes internal copies when necessary.  This means that sorting along an already contiguous axis may be faster.

Small changes like this don't get a release note since there is no information for someone updating NumPy (we auto generate a changelog with all PRs).

Reply:
You mean NumPy did not choose to sort only along a contiguous axis to improve performance; rather, it is impossible to sort along a non-contiguous axis?

Reply:
No, it's not impossible, but I doubt a concience choice was made that it is more performant, and it probably matters on the use-case if it is.

Anyway, I am OK with this also, I mainly thought it wasn't useful information (if correct).

Reply:
The low level  sort routines only handle contiguous data. I've sometimes thought of letting them handle strided data, but that could slow things down . There is also some history, the original work was done to replace the quicksort routine in numarray, and it took contiguous data.  Strided versions are out there, the NAG library, for instance, has a strided quicksort. I'm not sure what all the tradeoffs would be, they would probably be size and architecture dependent and vectorization might matter in newer versions.